### ISSUE TITLE
BN is train mode while Binary SSD_vgg16 is training ?

### ISSUE THREAD

**Issue by puhan123** [ZiweiWangTHU/BiDet#31]
Dear Author,

        I'm sorry to bother you .
        I noticed that the BN(Batch Norm) is train mode in the training process of SSD. The four parameters of BN (alpha, beta, meaning, variance) is update with the Binary neural network.
        Do I understand correctly?

       Thank you very much.

**Comment by Wuziyi616** [comment]
Yes, you are correct. This is a common practice in Binary Network training.
